bitcoin/src/blockdata/locktime/relative.rs

```diff
@@ -168,11 +168,11 @@ impl LockTime {
     /// # use bitcoin::Sequence;
     /// # use bitcoin::locktime::relative::{LockTime, Height, Time};
     ///
-    /// # let height = 100;       // 100 blocks.
+    /// # let required_height = 100;       // 100 blocks.
     /// # let intervals = 70;     // Approx 10 hours.
-    /// # let current_height = || Height::from(height + 10);
+    /// # let current_height = || Height::from(required_height + 10);
     /// # let current_time = || Time::from_512_second_intervals(intervals + 10);
-    /// # let lock = Sequence::from_height(height).to_relative_lock_time().expect("valid height");
+    /// # let lock = Sequence::from_height(required_height).to_relative_lock_time().expect("valid height");
     ///
     /// // Users that have chain data can get the current height and time to check against a lock.
     /// let height_and_time = (current_time(), current_height());  // tuple order does not matter.
```

Comment:
This variable appears unused. I think we should delete it. Also I've realized the example is unrealistically pretending that `current_*()` will not fail but presumably it's an rpc call.

Reply:
True, in real life these methods probably are something complicated and fallible, but the point of the example is to illustrate specifically the `Sequence` API so I think it's clearer to simplify everything else.

Reply:
I'd at least try to write it as `rpc_client.get_info()` to be obvious.

Reply:
The rpc call need not be anywhere near this code in a real project, right? The height values could be passed in as function paramaters then this snippet is usable. I think the lock times are complex enough without adding any additional complexity.